### PR TITLE
Remove LukasHirt from codeowners

### PR DESCRIPTION
**Pull request by LukasHirt**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1 @@
-* @LukasHirt
\ No newline at end of file
+
```
Comment (pascalwengerter) [inline .github/CODEOWNERS:1]:
```suggestion
@kulmann 
@pascalwengerter 
```

**Review by pascalwengerter — CHANGES_REQUESTED** [review]
Would like to avoid having this file empty :) thanks for taking care!

**Review by kulmann — APPROVED** [review]
😞

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1 +1,2 @@
-* @LukasHirt
\ No newline at end of file
+@kulmann
```
Comment (pascalwengerter) [inline .github/CODEOWNERS:1]:
```suggestion
* @kulmann 
```

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -1 +1,2 @@
-* @LukasHirt
\ No newline at end of file
+@kulmann 
+@pascalwengerter
```
Comment (pascalwengerter) [inline .github/CODEOWNERS:2]:
```suggestion
* @pascalwengerter 
```

**Review by pascalwengerter — COMMENTED** [review]
Sorry seems that I messed this one up 🤦🏽

**Review by pascalwengerter — APPROVED** [review]
Thanks 🙏🏽

**Pull request merged**